Source/Core/DolphinQt/Config/BBAConfigWidget.cpp

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt/Config/BBAConfigWidget.h"
+
+#include <QVBoxLayout>
+#include <QFormLayout>
+#include <QHBoxLayout>
+#include <QLineEdit>
+#include <QToolButton>
+#include <QLabel>
+#include <QSpinBox>
+#include <QDialogButtonBox>
+#include <QMessageBox>
+
+#include "Common/Network.h"
+
+BBAConfigWidget::BBAConfigWidget(bool showServer, QWidget* parent) :
+    QDialog(parent),
+    m_server(nullptr),
```

Comment:
These member variables can be initialized within the class declaration directly.